React-Homeworks/src/Pages/LoginPage/LoginPage.jsx

```diff
@@ -0,0 +1,85 @@
+import { signInWithEmailAndPassword } from "firebase/auth";
+import React, { useState } from "react";
+import auth from "../../firebase-config";
+import "./LoginPage.css";
+import Button from "../../components/Buttons/Buttons";
+
+const SignIn = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [isLogged, setIsLogged] = useState(null);
```

Comment:
You store user status in this component it will prevent you to know the status after this component will be unmounted